README.md

```diff
@@ -106,6 +109,46 @@ We use the basic feature branch GIT flow. Create a feature branch off of master
 request. Make branch names and commits descriptive. A merge request should solve one problem,
 not many. 
 
+## Offline installation
```

Comment:
move this section to the bottom of the README (just above the versions section)

Reply:
54732611b43cdeecf556daa1c4c6421dce5c7d73